app/javascript/bundles/RestaurantsApp/components/AddRestaurant.jsx

```diff
@@ -62,6 +65,7 @@ export default class AddRestaurantForm extends React.Component {
             };
 
             const data = JSON.stringify(restaurant);
+            const onAdd = this.props.onAdd;
             const onClose = this.props.onClose;
```

Comment:
Check this out: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

TL;DR:
```js
const {
  onAdd,
  onClose,
} = this.props;
```